src/main/java/me/lokka30/levelledmobs/customdrops/CustomDropsParser.java

```diff
@@ -341,6 +341,11 @@ else if (itemEntry.getValue() instanceof ArrayList)
 
     private void parseCustomDropsAttributes(@NotNull final CustomDropBase dropBase, @NotNull final ConfigurationSection cs, final @NotNull CustomDropInstance dropInstance){
         dropBase.chance = ymlHelper.getFloat(cs, "chance", this.defaults.chance);
+        dropBase.deathInChunkThreshold = ymlHelper.getInt(cs, "deathInChunkThreshold", this.defaults.deathInChunkThreshold);
```

Comment:
In configs we use formatting like `death-in-chunk-threshold` instead of `camelCase` (used for variables of course). :smiley: